CHANGES.md

```diff
@@ -1,5 +1,88 @@
+ - Add coverage via `codecov`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3463)
+
+**BUG**
+
+ - Add utility function to strip '.py' extensions.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3658)
+ - **sct_maths:** Appply -symmetrize on specified dimension in `sct_maths`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3657)
+ - **sct_compute_mtr:** Convert input file to float32 before compute MTR.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3638)
+ - **sct_get_centerline:** Make sure that the QC report for `sct_get_centerline` displays properly.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3626)
+ - Set `channel_axis=None` for `pyramid_expand` to fix bug in `skimage==0.19.0`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3616)
+ - **sct_propseg:** Fix argument parsing to ensure only the requested files are output.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3611)
+ - Restore the argparse linewrapping behavior we had prior to Python 3.7.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3604)
+ - **sct_download_data:** Update outdated mirror links.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3596)
+ - Upgrade to Qt5..  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3588)
+ - **sct_resample:** Make sure dimensions are correct for 4D images resampled with `-mm`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3553)
+ - **sct_qc:** Fix colormap for vertebral labeling in the QC report.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3534)
+
+**INSTALLATION**
+
+ - Allow `install_sct` to run from any directory.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3660)
+ - Move dependencies from `requirements.txt` to `setup.py :: install_requires`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3646)
+ - Use `--ignore-installed` to preserve the version of `certifi` that gets installed during `conda create`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3618)
+ - Use a more reliable way to disable USER_SITE.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3606)
+ - `requirements.txt`: Pin `onnxruntime>=1.7.0` rather than `==1.4.0`.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3595)
+ - `install_sct`: Skip `pip==21.2` to avoid suboptimal dependency resolver.  [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3594)
```

Comment:
Maybe we should add a light-purple `install_sct` label? I noticed that for the _Installation_ and _Git/Github_ categories, there isn't always an applicable light-purple label.

And if not, in the future I'll try to use pseudo-light-purple labels like the linked PR.

Reply:
Currently, light purple labels only really have functionality re: prepending `sct_<script>` to relevant changelog items. 

Apart from that, it's really just organizational on GitHub? (And, part of me suspects that labels aren't used by anybody to filter PRs...) 

So, while I definitely think it makes sense in theory, I'm not sure if it would have a meaningful effect (unless we also went and modified our [`changelog`](github.com/neuropoly/changelog/) library? Hmm...).

Reply:
Truthfully, some of the distinctions between dark purple and light purple feel a bit off at times? But I've shrugged my concerns off, because I think back to trying to accurately catalog my music library genres and remember that categorization is a hard problem, ehe.

I think my preferred approach is to say, "Which label scheme generates the nicest looking changelog?" --> i.e. looking at the changelog and working backwards.